src/page-modules/assistant/types.ts

```diff
@@ -15,12 +15,17 @@ export type TripInput = {
   cursor?: string;
 };
 
+export type TripViaInput = TripInput & {
+  via: GeocoderFeature;
+};
+
```

Comment:
If my other comment makes sense to you, we could change the `type TripInput` to 

```ts
type TripInput = {
from: GeocoderFeature;
to: GeocoderFeature;
via?: GeocoderFeature;
...
}
```

I think the question comes down to: is there a difference between a trip that has a via location and a trip that does not?